source/includes/gcp/_ssl_certificates.md

```diff
@@ -166,3 +166,17 @@ Create a new ssl certificate
 | `description`<br/>*string* | A short description of this certificate. |
 | `name`<br/>*string* | The display name of the ssl certificate |
 | `privateKey`<br/>*string* | The privateKey associated to certificate in PEM format. |
+
+<!-------------------- DELETE A SSL CERTIFICATE -------------------->
+
+##### Delete a ssl certificate
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/sslcertificates/ssl-root-rnx"
```

Comment:
This should be the certificate ID, not its name.

Reply:
Done.